WeWatch/WeWatch/Loader/LoaderView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  LoaderView.swift
+//  WeWatch
+//
+//  Created by Anton on 22/12/2024.
+//
+
+import SwiftUI
+import Lottie
+
+extension View {
+    
+    internal func loader(isLoading: Bool) -> some View {
+        ZStack {
+            VStack{
+                LottieView(animation: .named("loader"))
+                    .playbackMode(.playing(.toProgress(1, loopMode: .loop)))
+                Text("loader.label.title")
+                Spacer(minLength: 20)
+            }
+            .padding(.horizontal, 24)
+            .frame(maxWidth: .infinity, maxHeight: 142)
+            .foregroundColor(.whiteColor)
+            .font(.poppinsRegular16px)
+            .background(RoundedRectangle(cornerRadius: 12.0).fill(Color.darkGreyColor))
```

Comment:
I think your padding isn’t working as intended. 😄 Please double-check it and attach a screenshot so I can verify that your loader looks as expected.